### PR TITLE
Add tupleToArray point free function

### PR DESCRIPTION
**Pull request by jasminabasurita**
### Changes:
  - added a `tupleToArray` point free function
  - added checks to ensure the passed argument is a Tuple or a Tuple returning function

### CODE REVIEW

File: docs/src/pages/docs/crocks/Tuple.md [hunk 1]
```diff
@@ -474,3 +474,40 @@ trimap(add(10), toUpper, add(10), triple)
...
+
+import tupleToArray from 'crocks/Tuple/tupleToArray'
+
+const Triple = Tuple(3)
+
+const triple = Triple( 1, { key: 'value' }, 'string' )
+
+tupleToArray(triple) // => [ 1, { key: 'value' }, 'string' ]
+
+tupleToArray(() => triple)() // => [ 1, { key: 'value' }, 'string' ]
```
Comment (dalefrancis88) [inline docs/src/pages/docs/crocks/Tuple.md]: Not sure i like this as an example as the function used is impure, but it's so simple i might wait for @evilsoft to chime in.

File: src/Tuple/index.js [hunk 2]
```diff
@@ -22,6 +22,8 @@ function _Tuple(n) {
     throw new TypeError('Tuple: First argument must be an integer')
   }
 
+  const tupleLength = () => n
```
Comment (dalefrancis88) [inline src/Tuple/index.js]: this could be constant(n) to be pure

File: docs/src/pages/docs/crocks/Tuple.md (hunk 1 above)
Comment (jasminabasurita) [reply docs/src/pages/docs/crocks/Tuple.md]: perhaps this:
`tupleToArray(identity)(triple) // => [ 1, { key: 'value' }, 'string' ]`

**Comment by dalefrancis88** [comment]
Looks like no further comments, and looks good. 
image

**Comment by dalefrancis88** [comment]
On a side note, you should join our Gitter. Great place to give and receive help.
https://gitter.im/crocksjs/crocks

**Comment by jasminabasurita** [comment]
:tada: :unicorn: :heavy_heart_exclamation:

**Pull request merged**